back/db/structure.sql

```diff
@@ -1845,13 +1830,10 @@ CREATE VIEW public.analytics_fact_posts AS
     i.likes_count,
     i.dislikes_count,
     i.publication_status
-   FROM (((((public.ideas i
+   FROM (((public.ideas i
      JOIN public.analytics_dimension_types adt ON (((adt.name)::text = 'idea'::text)))
      LEFT JOIN public.analytics_build_feedbacks abf ON ((abf.post_id = i.id)))
-     LEFT JOIN public.ideas_phases iph ON ((iph.idea_id = i.id)))
-     LEFT JOIN public.phases ph ON ((ph.id = iph.phase_id)))
      LEFT JOIN public.projects pr ON ((pr.id = i.project_id)))
-  WHERE ((ph.id IS NULL) OR ((ph.participation_method)::text <> 'native_survey'::text))
```

Comment:
This is where you can see the change to the `fact_posts` view. Before it was checking that the idea had a phase or it was a native_survey response. But by doing so it was joining `ideas_phases`, so where there were multiple phases this view then had multiple rows per phase.

I took the view that actually it doesn't matter if the idea has a phase, it's still an idea on a project and therefore should be counted, so I removed the clause entirely - esp as other views such as `fact_participations` do not do this check.